RevenueCatUI/Data/Strings.swift

```diff
@@ -98,6 +103,22 @@ extension Strings: CustomStringConvertible {
 
         case .setting_restored_customer_info:
             return "Setting restored customer info"
+
+        case .executing_external_purchase_logic:
+            return "Will execute custom StoreKit purchase logic provided by your app. " +
+            "No StoreKit purchasing logic will be performed by RevenueCat. " +
```

Comment:
This seems a bit misleading to me - a _lot_ of StoreKit purchasing logic will be performed by RevenueCat regardless. We still fetch products, post receipts, etc, right? 
The only real change is that we won't finish transactions. I realize that that might not be a concept that a user grasps all that well, but then again, we're assuming that this developer has a full StoreKit implementation of their own, so it feels reasonable that they could figure out what finishing transactions is?

Reply:
How about something like

> Initiating a purchase for product <product_id>. Note that the StoreKit purchasing logic will be provided by your app and not RevenueCat, since the SDK was configured to blah blah blah.

Reply:
I think my confusion here is regarding exactly what StoreKit logic is outsourced to our customer's app.

This:
> Will execute custom StoreKit purchase logic provided by your app. No StoreKit purchasing logic will be performed by RevenueCat.

sounds the same to me as:
> the StoreKit purchasing logic will be provided by your app and not RevenueCat

The only real change is that we won't finish transactions, but when `purchasesAreCompletedBy` is `.myApp`, we don't call into `PurchasesOrchestrator.purchase(product:package:completion:)` - is this part of finishing transactions?

I think what matters most is that someone who has written or is going to write their own StoreKit interfacing code understands exactly what they are now responsible for. Should we list the StoreKit API calls we no longer call? I'm a bit concerned that unless we're crystal clear, there may be confusion on the customer's side too.

Reply:
Yeah, that was my specific concern as well - we need to make it clear that you need to finish transactions. 
That's really the only important part. 

Finishing transactions happens specifically when we _detect_ a purchase, then post it to the backend, and receive a response from the backend. 

So even if you don't go through the purchase method, we can still detect the purchase, and that's when we finish transactions (unless this mode is enabled). 

It'd be [finishTransaction](https://developer.apple.com/documentation/storekit/skpaymentqueue/1506003-finishtransaction#) in SK1, [transaction.finish()](https://developer.apple.com/documentation/storekit/transaction) in SK2